manifests/local.pp

```diff
@@ -22,11 +22,9 @@
 
   # Since there already are local audispd audit logs in /var/log/audit and these
   # logs grow quickly, drop the syslog duplicates.
-  rsyslog::rule::local { "${order}1_simp_rsyslog_profile_local_drop_audispd_duplicates":
-    rule       => '$programname == \'audispd\'',
-    dyna_file  => '~',
-    # We don't need this due to the line above
-    stop_processing => true
+  $_safe_order = regsubst($order,'/','__')
+  rsyslog::rule::local {"${_safe_order}1_simp_rsyslog_profile_local_drop_audispd_duplicates.conf":
```

Comment:
When you change the fixtures file can you change the name of this removing the .conf.  rsyslog adds the .conf on and it is called simp_rsyslog_profile_local_drop_audispd_duplicates.conf.conf